compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -27,6 +27,9 @@
  */
 public enum DiagnosticErrorCode implements DiagnosticCode {
 
+    // The member represents a bad sad error
```

Comment:
Shall we add a comment here explaining that this is a special diagnostic code.

Reply:
Updated in 1bfe3d6700a474472e3d57098ca04df8126d908d